docs/src/main/paradox/java/http/routing-dsl/directives/security-directives/index.md

```diff
@@ -20,7 +20,7 @@
 
 @@@
 
-<a id="authentication-vs-authorization-java"></a>
+<a id="authentication-vs-authorization"></a>
```

Comment:
Do we still need those at all?

Reply:
only when they don't correspond 1-to-1 with the heading I believe... would be nice to go through those but then I'd like to have https://github.com/lightbend/paradox/issues/120 first

Reply:
I see, makes sense.